application/src/main/java/uk/nhs/hee/tis/revalidation/controller/DoctorsForDBController.java

```diff
@@ -97,11 +99,12 @@ public ResponseEntity<TraineeSummaryDto> getTraineeDoctorsInformation(
       @RequestParam(name = UNDER_NOTICE, defaultValue = UNDER_NOTICE_VALUE, required = false) final boolean underNotice,
       @RequestParam(name = PAGE_NUMBER, defaultValue = PAGE_NUMBER_VALUE, required = false) final int pageNumber,
       @RequestParam(name = DESIGNATED_BODY_CODES, required = false) final List<String> dbcs,
-      @RequestParam(name = PROGRAMME_NAME_PARAM, required = false) final String programmeName,
+      @RequestParam(name = PROGRAMME_NAME_PARAM, defaultValue = EMPTY_STRING, required = false) final String programmeName,
+      @RequestParam(name = GMC_STATUS_PARAM, defaultValue = EMPTY_STRING, required = false) final String gmcStatus,
       @RequestParam(name = SEARCH_QUERY, defaultValue = EMPTY_STRING, required = false) final String searchQuery) {
```

Comment:
Supplying `defaultValue` negates the need for `required = false`. See https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/RequestParam.html#required--